shop/src/main/java/ro/msg/learning/shop/domain/Order.java

```diff
@@ -0,0 +1,36 @@
+package ro.msg.learning.shop.domain;
+
+import jakarta.persistence.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.jpa.convert.threeten.Jsr310JpaConverters;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Entity
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class Order extends EntityWithUUID {
+    @ManyToOne
+    @MapsId("costumer")
+    @JoinColumn(name = "id")
+    private UUID customerId;
```

Comment:
Here you should have a field `private Customer customer` (based on the foreign key you can load the whole customer object if needed). An entity should not be linked to other entity IDs, but to other entity objects.